### DISCUSSION TITLE
Linux: Microphone, Muting, Screensharing not working

### DISCUSSION THREAD

**Discussion by Ssionn** [zed-industries/zed#14081] [Feature Requests]
Microphone, Muting, Screensharing isn't working on the released Linux build. Anyway to fix this?
![image](https://github.com/zed-industries/zed/assets/90753599/9cd55a38-4328-46b5-b261-998bdb53832a)

**Comment by SomeoneToIgnore** [comment]
It does not indeed, and needs https://github.com/zed-industries/zed/pull/13343 to be merged and released first.
